src/main/java/com/github/forax/umldoc/editor/MermaidCmdLineParser.java

```diff
@@ -0,0 +1,69 @@
+package com.github.forax.umldoc.editor;
+
+import com.github.forax.umldoc.gen.MermaidGenerator;
+import java.util.Optional;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+
+/**
+ * This class is used to parse the command line for the mermaid export.
+ *
+ */
+public class MermaidCmdLineParser implements CommandLineParser {
+  private boolean hasGenerationStarted = false;
```

Comment:
It's not necessary, the code that calls that interface already know if it is inside of outside the %%umldoc block